addons/mail/static/src/models/notification_group/notification_group.js

```diff
@@ -97,23 +126,63 @@ function factory(dependencies) {
     }
 
     NotificationGroup.fields = {
-        date: attr(),
+        /**
+         * Determine the most recent date of all the notification message.
```

Comment:
Determine**s** if we follow conjugation of other doc-strings.

But in this case I think it should use the verb States instead, because Determines is used to indicate the field could be updated to change the value, but in this case it is a compute.